### PR TITLE
Application system - mobile refinements 1

### PR DESCRIPTION
**Pull request by madebynoam**
Various mobile refinements and fixes for the application system:

1. Fix cases where a horizontal scroll bar appears
2. Update Continue and Back buttons to match latest designs
3. Fix logo not appearing
4. Turn off auto focus fields
5. Scroll to top on question transition
6. Make the stepper sticky on mobile size
7. Make the stepper move to the active section on mobiles size
8. Wrap the the table with `overflowX: scroll` so that it doesn't create horizontal scroll bars for the main shell.
9. Start setting field types to more semantic attributes (for the relevant mobile keyboards).

Remaining (for next PR):
1. Make chapter images full width on mobile size
2. Add a sub-section progress bar, since the stepper only shows main sections on mobile size.
3. Continue investigating how to do `escapedGrid` blue containers.

## Screenshots / Gifs

application-shell-mobile

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by jeremybarbet — APPROVED** [review]
LGTM

File: libs/island-ui/core/src/lib/FormStepper/FormStepperSection.tsx [hunk 1]
```diff
@@ -40,57 +41,92 @@ export const FormStepperSection: FC<{
...
+    }
+
+    handleResize()
+
+    window.addEventListener('resize', handleResize)
+
+    return () => {
+      window.removeEventListener('resize', handleResize)
+    }
+  }, [, isClient])
```
Comment (arnifreyrs) [inline libs/island-ui/core/src/lib/FormStepper/FormStepperSection.tsx:86]: do you need to have empty first element in the dependency array?

File: libs/application/ui-shell/src/components/FormMultiField.tsx [hunk 2]
```diff
@@ -40,7 +40,6 @@ const FormMultiField: FC<{
                 showFieldName
                 field={field as FieldDef}
                 key={field.id}
-                autoFocus={index === 0}
```
Comment (arnifreyrs) [inline libs/application/ui-shell/src/components/FormMultiField.tsx:43]: should we remove `autoFocus` from props in `FormField` as well?

**Review by arnifreyrs — COMMENTED** [review]
Awesome stuff! My comments are just minor and something you could consider fixing in the next PR

**Pull request merged**